designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/SupportingText.kt

```diff
@@ -132,29 +143,43 @@ fun SupportingText(
         }
     }
 
+    val density = LocalDensity.current
+    val visibleState = remember {
```

Comment:
how does this react with multiple supporting texts? We have the option to set single or multiple supporting texts in components. Wouldn't this animate each supporting text sliding from the top vertically? Can you confirm with Marcos whether that's the expected behaviour?